README.md

```diff
@@ -1,4 +1,4 @@
-[![slack](https://img.shields.io/badge/slack-argoproj-brightgreen.svg?logo=slack)](https://argoproj.github.io/community/join-slack)
+[![slack](https://img.shields.io/badge/slack-argo_workflows-brightgreen.svg?logo=slack)](https://cloud-native.slack.com/archives/C01QW9QSSSK)
```

Comment:
@alexmt can we have a page on argoproj.io that captures analytics and then redirects please?

Reply:
That would be great. Let me know once it's ready and I can update the link here.